local-cli/runAndroid/runAndroid.js

```diff
@@ -123,15 +123,28 @@ function buildAndRun(args, reject) {
 }
 
 function startServerInNewWindow() {
+  var yargV = require('yargs').argv;
+
   const launchPackagerScript = path.resolve(
     __dirname, '..', '..', 'packager', 'launchPackager.command'
   );
 
   if (process.platform === 'darwin') {
-    child_process.spawnSync('open', [launchPackagerScript]);
+    if (!yargV.open) {
```

Comment:
I think something like this would be more readable & maintainable if we change the command:

```js
const termApp = yargV.open ? yargV.open : 'Terminal.app';
return child_process.spawnSync('open', ['-a', termApp, launchPackagerScript]);
```